api/src/main/java/com/github/retrooper/packetevents/protocol/player/User.java

```diff
@@ -361,4 +369,52 @@ public NBTCompound getWorldNBT(String worldName) {
         String dimensionName = dimension.getDimensionName();
         return dimensionName.isEmpty() ? this.getWorldName(dimension.getId()) : dimensionName;
     }
+
+    /**
+     * Stores the given {@param value} with the given {@param identifier}.
+     * This essentially makes this User object associated with the stored value.
+     * Enables a quick look-up of the associated value with the {@link #getStored(StorageValueId)}
+     * method.
+     *
+     * @param identifier The plugin's constant identifier created from {@link StorageValueId#identifierFor StorageValueId.identifierFor}
+     * @param value The value to store.
+     */
+
+    public void storeRuntime(StorageValueId identifier, Object value) {
```

Comment:
i think just `store` would be sufficient for this method name

Reply:
I thought it would remove any confusion from this being a persistent store